src/plugins/data_table/data_table.js

```diff
@@ -349,6 +350,17 @@ var LocalDataTable = (function() {
       this.dataTable.on("filter", this._bulkCheckboxAdjust);
     },
 
+    _enableRowHighlight: function() {
+     this.dataTable.on("click", this.ROWS_SELECTOR, this._onRowHighlightClick);
```

Comment:
Please use a 2 spaces indentation here. The same for `_onRowHighlightClick` method below.